Dockerfile-bcoin

```diff
@@ -1,30 +1,48 @@
 FROM nfnty/arch-mini:latest AS base
 
-RUN mkdir -p /code/node_modules/bcoin /data
-WORKDIR /code/node_modules/bcoin
-ENTRYPOINT ["node"]
-CMD ["/code/scripts/docker-bcoin-init.js"]
-
 RUN pacman -Sy --noconfirm archlinux-keyring && \
     pacman -Syu --noconfirm nodejs npm && \
     rm /var/cache/pacman/pkg/*
 
 FROM base AS build
 # Install build dependencies
-RUN pacman -Syu --noconfirm base-devel unrar git python2
-#HACK: Node-gyp needs python
-RUN ln -s /usr/bin/python2 /usr/bin/python
+# Note: node-gyp needs python
+RUN pacman -Syu --noconfirm base-devel unrar git python2 \
+    && ln -s /usr/bin/python2 /usr/bin/python
 
-ARG repo=https://github.com/bcoin-org/bcoin.git
 ARG branch=master
-#HACK: Dingle this to trigger rebuilds
+
+# use this to bust the build cache
 ARG rebuild=0
 
 # Install bcoin
-RUN git clone --branch $branch $repo /code/node_modules/bcoin
-RUN npm install --production
+RUN git clone --branch $branch \
+    https://github.com/bcoin-org/bcoin.git /code/node_modules/bcoin \
+    && pushd /code/node_modules/bcoin \
+    && npm install --production \
+    && popd
+
+# Install bmultisig
+RUN git clone --branch $branch \
+    https://github.com/bcoin-org/bmultisig.git /code/node_modules/bmultisig \
+    && pushd /code/node_modules/bmultisig \
+    && npm install --production \
+    && popd
+
```

Comment:
Comment indicating this installs bledger (could be more in the future, e.g. trezor, not strictly necessary for multisig)

Reply:
What do you mean by that?

I don't see `bledger` in `bmultisig/package.json` - https://github.com/bcoin-org/bmultisig/blob/master/package.json

Reply:
My bad, misread 😊